code/game/machinery/vending/vendor_types/crew/commanding_officer.dm

```diff
@@ -26,6 +27,8 @@ GLOBAL_LIST_INIT(cm_vending_gear_commanding_officer, list(
 		list("SPECIAL AMMUNITION", 0, null, null, null),
 		list("M41A Incendiary Magazine", 65, /obj/item/ammo_magazine/rifle/incendiary, null, VENDOR_ITEM_REGULAR),
 		list("M41A Rubber Shot Magazine", 10, /obj/item/ammo_magazine/rifle/rubber, null, VENDOR_ITEM_REGULAR),
+		list("M42A Incendiary Magazine", 45, /obj/item/ammo_magazine/sniper/incendiary, null, VENDOR_ITEM_REGULAR),
+		list("M42A Flak Magazine", 30, /obj/item/ammo_magazine/sniper/flak, null, VENDOR_ITEM_REGULAR),
```

Comment:
price diff seems somewhat arbitrary, considering these three ammo types are very similar in power

Reply:
by this I meant make them all 30 points or whatever, marksman is as strong as flak due to aimed shot